pandas/core/tools/timedeltas.py

```diff
@@ -242,17 +244,23 @@ def _coerce_scalar_to_timedelta_type(
 
 
 def _convert_listlike(
-    arg, unit=None, errors: DateTimeErrorChoices = "raise", name=None
+    arg,
+    unit: UnitChoices | None = None,
+    errors: DateTimeErrorChoices = "raise",
+    name: Hashable | None = None,
```

Comment:
i think Hashable includes None?

Reply:
mypy doesn't complain when removing None, but looks like there's a ruff check that auto-annotates `None` here